test/perf/e2e.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2020 VMware, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+package perf
```

Comment:
Is the idea here to isolate the perf package from the e2e?
Personally, I would prefer to have these common structs at the test package level and be consumed by both sets of tests.

Reply:
Ok, i'll see if I can move `imgpkg` and the `logger` somewhere more re-usable between the two kind of test suites

Reply:
I decided to move every utility file (even those that are not shared across the 2 packages) into a 'helpers' package.

It results in only test files living under e2e / perf, which I think is cleaner because it helps to see what is being tested at a glance.